### DISCUSSION TITLE
Welcome to cockpit Discussions!

### DISCUSSION THREAD

**Discussion by martinpitt** [cockpit-project/cockpit#15755] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by garrett** [comment]
Hi all! I'm Garrett, and I'm a designer + developer working on Cockpit!

I've been in the FOSS community for decades (since 1996, shortly after discovering Linux and wanting to "give back"), working on stuff from Linux.com and Themes.net back in the late 90s to early 2000s at VA Linux / Software / etc. and OSDN. Then I joined Red Hat for the firs time around, made Bluecurve (to unify the look and feel across all of Red Hat Linux & RHEL), and handled a bunch of UI design of various projects (like Red Hat Network with up2date), and started Fedora with a small team. Afterward, I went on to work for Novell / SUSE, where I worked on the Linux desktop for a while (mainly with a focus on GNOME in the 2.x and early 3.x days), started the Tango Project to make icon standards and a reference icon set (and color palette), was a part of BetterDesktop (open source usability testing) with a few others, and then worked on SUSE Studio for a while.

After all that, I rejoined Red Hat in the OSAS group (which later became OSPO) and eventually made my way to Cockpit.

During all this time, I've worked on tons of different projects and have kept in touch with web standards (especially HTML and CSS, but also a bit of JavaScript too).

**TL;DR**: I'm a coding designer who has been working to make FOSS better for decades. :wink:
